gulp/docs.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var gulp = require('gulp');
+var shell = require('gulp-shell');
+
+gulp.task('docs', shell.task([
+    'mkdir "dist/docs" \\' +
```

Comment:
this no longer needed if you remove the ` '-u dist/docs/tutorials'   // tutorials `directory line